myapp/playlist.js

```diff
@@ -0,0 +1,88 @@
+var parsedPlaylist = JSON.parse(playlistsJSON);
+var l = parsedPlaylist.length;
+
+var playlist = [];
+var songs = [];
+
+
+
+function Playlist(data){
+  this.id = data.id;
+  this.title = data.title;
+  this.songs = [];
+  for( j = 0; j < data.songs.length; j++)
+  {
+    var tempSong = new Song(data.songs[j]);
+    this.songs.push(tempSong);
+  }
+}
+
+
+for(i = 0; i < l; i++){
+  playlist[i] = new Playlist(parsedPlaylist[i]);
+}
+
+
+Playlist.prototype.setId = function (id){
+    this.id = id;
+};
+Playlist.prototype.setTitle = function (title){
+    this.title = title;
+};
+Playlist.prototype.setSongs = function (songs){
+    this.songs = songs;
+};
+
+Playlist.prototype.getId = function (){
+    return this.id;
+};
+Playlist.prototype.getTitle = function (){
+    return this.title;
+};
+Playlist.prototype.getSongs = function (){
+    return this.songs;
+};
+
+
+function removePlaylist(){
```

Comment:
separate these 2 functions, which do  DOM access into their own file( don't put them together with the bussines logic ).